main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"github.com/mp-hl-2021/code-swamp/api"
+	"github.com/mp-hl-2021/code-swamp/usecases"
+	"net/http"
+	"time"
+)
+
+func main() {
+	user := &usecases.User{}
+
+	service := api.NewApi(user)
+
+	server := http.Server{
+		Addr:         "localhost:8080",
+		ReadTimeout:  10 * time.Second,
+		WriteTimeout: 10 * time.Second,
+
+		Handler: service.Router(),
+	}
+	err := server.ListenAndServe()
+	if err != nil {
+		panic(err)
+	}
```

Comment:
It might be good to display some initial greeting from your running server here.